js/index.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function() {
+
+$('.read-more').click(readMore);
+	function readMore () {
```

Comment:
since you're not embedding this click function (`readMore`) inside the `click`'s parenthesis, you actually don't need to indent the function.

In other words - do this:

```
$('.read-more').click(readMore);
function readMore () {
     $('#readMoreText').slideDown();
     $('.read-less').show();
     $('.read-more').hide();
}
```
alternatively, you could do this (but I prefer the one above):

```
$('.read-more').click(function(){
     $('#readMoreText').slideDown();
     $('.read-less').show();
     $('.read-more').hide();
});
```